packages/next-safe-action/src/hook.ts

```diff
@@ -163,27 +166,23 @@ export const useOptimisticAction = <const Schema extends z.ZodTypeAny, const Dat
 			setOptimisticState(input);
 			setInput(input);
 
-			return executor
+			executor
 				.current(input)
-				.then((res) => setResult(res))
+				.then((res) => (res ? setResult(res) : void 0))
 				.catch((e) => {
-					// NOTE: this doesn't work at the moment.
+					// TODO: this doesn't work at the moment.
```

Comment:
This is a NOTE and not a TODO comment because the issue is on Next.js end, and not with this library. Is this actually fixed now? If so, it should be removed.